src/data/actions/stafferInfo.test.js

```diff
@@ -0,0 +1,36 @@
+import {
```

Comment:
It would be great to get some more tests for the `stafferInfo` actions

Reply:
✅